sdk/keyvault/azure-security-keyvault-administration/README.md

```diff
@@ -14,7 +14,7 @@ Maven dependency for the Azure Key Vault Administration library. Add it to your
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-administration</artifactId>
-    <version>4.0.0-beta.5</version>
+    <version>4.0.0-beta.6</version>
```

Comment:
Is this version released?

Reply:
We are releasing these versions. I updated the READMEs so they would be mentioned in the generated docs.

Reply:
I will update to the GA versions after the beta release. Last month I forgot to do that but that's how it should look like.